### PR TITLE
Create models course topic article

### PR DESCRIPTION
**Pull request by katherinameliuk**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -104,7 +104,7 @@ celerybeat.pid
 *.sage.py
 
 # Environments
-.env
+.env.dev
```
Comment (DiStan92) [inline .gitignore]: delete .dev

File: EDUPlatform/users/models.py [hunk 2]
```diff
@@ -39,21 +39,61 @@ class Teacher(User):
...
+    course_name = models.CharField(max_length=50)
+    teacher = models.ForeignKey(Teacher, on_delete=models.CASCADE)
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+
+    def __str__(self):
+        return f"{self.course_name}, {self.teacher}, {self.price}"
+
+    class Meta:
+        verbose_name = "course"
+        verbose_name_plural = "course"
```
Comment (DiStan92) [inline EDUPlatform/users/models.py:73]: verbose_name_plural = "courses"

File: EDUPlatform/users/models.py [hunk 3]
```diff
@@ -39,21 +39,61 @@ class Teacher(User):
...
+    topic_name = models.CharField(max_length=100)
+    course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    image = models.TextField()
+
+    def __str__(self):
+        return f"{self.topic_name}, {self.course}"
+
+    class Meta:
+        verbose_name = "topic"
+        verbose_name_plural = "topic"
```
Comment (DiStan92) [inline EDUPlatform/users/models.py:86]: "topics"

File: EDUPlatform/users/models.py [hunk 4]
```diff
@@ -39,21 +39,61 @@ class Teacher(User):
...
+
+    def __str__(self):
+        return f"{self.title}"
 
     class Meta:
-        verbose_name = 'student'
-        verbose_name_plural = 'students'
-        ordering = ['first_name', 'last_name']
+        verbose_name = "article"
+        verbose_name_plural = "article"
```
Comment (DiStan92) [inline EDUPlatform/users/models.py:100]: "articles"

File: EDUPlatform/users/models.py [hunk 5]
```diff
@@ -39,21 +39,61 @@ class Teacher(User):
...
+
+    class Meta:
+        verbose_name = "course"
+        verbose_name_plural = "course"
+
+
+class Topic(models.Model):
+    topic_name = models.CharField(max_length=100)
+    course = models.ForeignKey(Course, on_delete=models.CASCADE)
+    image = models.TextField()
```
Comment (nikitapt63) [inline EDUPlatform/users/models.py:79]: image cannot be in TextField, use ImageField or FileField

File: EDUPlatform/users/models.py [hunk 6]
```diff
@@ -39,21 +39,61 @@ class Teacher(User):
...
+        return f"{self.topic_name}, {self.course}"
+
+    class Meta:
+        verbose_name = "topic"
+        verbose_name_plural = "topic"
+
+
+class Article(models.Model):
+    title = models.CharField(max_length=150)
+    topic = models.ForeignKey(Topic, on_delete=models.SET_NULL, null=True)
```
Comment (nikitapt63) [inline EDUPlatform/users/models.py:91]: why set_null?

**Review by nikitapt63 — CHANGES_REQUESTED** [review]
One more thing, i think its better to add Course, Topic and Article in testing_system.models)

**Review by nikitapt63 — APPROVED** [review]

**Pull request merged**